docs/concepts/python-versions.md

```diff
@@ -142,7 +131,7 @@ This will install a Python executable for the requested version into `~/.local/b
 To install `python` and `python3` executables, include the `--default` option:
 
 ```console
-$ uv python install 3.12 --default --preview
+$ uv python install 3.12 --default
```

Comment:
```suggestion
uv python install 3.12 --default --preview
```

Reply:
Didn't mean to remove the `$`. Locally, I still get the warning that `--default` requires preview, my understanding reading #14681 was that that was intentional?

```
$ cargo run -q python install 3.12 --default
warning: The `--default` option is experimental and may change without warning. Pass `--preview` to disable this warning
```

If it was intentional that the warning is shown when using `uv python install 3.12 --default` after #14681 cause we want it to work and be documented without `--preview` enabling other things like the upgrades, than that's fine too.